sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala

```diff
@@ -805,6 +806,75 @@ class StreamSuite extends StreamTest {
     }
   }
 
+  test("streaming limit without state") {
```

Comment:
Related to my above comment, I think all of these tests end up only testing a single input partition.

Reply:
resolved